meteor-app/imports/ui/pages/workspace/style.less

```diff
@@ -75,9 +75,12 @@
       .map-animation-controls {
         .field--year {
           .flex-row;
+          align-items: center;
```

Comment:
Maintain a blank line to visually separate modules (e.g. `.flex-row`) and normal styles.